requirements/pytorch/strategies.txt

```diff
@@ -1,3 +1,4 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 deepspeed>=0.6.0, <=0.8.0
```

Comment:
Do you think the deepspeed issue you saw would also impact our users? If yes, we should also do

```suggestion
deepspeed>=0.6.0, <0.8.0
```